package.json

```diff
@@ -16,23 +16,43 @@
     "graphql-tag": "2.10.4",
     "next": "9.4.4",
     "react": "16.13.1",
-    "react-dom": "16.13.1"
+    "react-dom": "16.13.1",
+    "react-feather": "2.0.8",
```

Comment:
Just putting a comment that came to mind but it's not blocking for this PR.
I'm not against `react-feather` in principle but in the future, it would be great to get rid of it.

Here is my thinking: we need to manage a repo that will be forked by other people that will add their design systems on top of it. It would then be great to not "impose" a dependency that these other system would use.

But then again, this PR is already big so let's not do it now ^^